src/full_node/mempool_manager.py

```diff
@@ -317,11 +317,23 @@ async def add_spendbundle(
                 continue
 
             # Verify aggregated signature
-            validates = AugSchemeMPL.aggregate_verify(
```

Comment:
Why not avoid increasing indentation by inserting the error case up front like this:

if len(msgs) == 0 and new_spend.aggregated_signature != G2Element():
                        return (
                        None,
                        MempoolInclusionStatus.FAILED,
                        Err.BAD_AGGREGATE_SIGNATURE,
                    )